### PR TITLE
Doc install docs

### PR DESCRIPTION
**Pull request by tacaswell**
Windows and mac devs should make sure I have not introduced any false hoods.

Attn @matthew-brett to make sure I have not lied about wheels.

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -19,52 +19,25 @@ Installing
...
-<https://www.enthought.com/products/canopy/>`_ are both excellent choices that
-"just work" out of the box for Windows, macOS and common Linux platforms.  Both
-of these distributions include Matplotlib and *lots* of other useful tools.
-
-Linux : using your package manager
-----------------------------------
-
-If you are on Linux, you might prefer to use your package manager.  Matplotlib
-is packaged for almost every major Linux distribution.
+   The following backends work out of the box: Agg, TkAgg, ps, pdf and svg.
```
Comment (choldgraf) [inline INSTALL.rst]: this in particular seems important...are we testing each of these backends in the tests so we can be sure this is true?

**Comment by choldgraf** [comment]
I re-triggered the docs build on circle 3.5, since I *think* it's related to a temporary internet issue...

**Comment by QuLogic** [comment]
Nope; you'll need to rebase to fix the 404.

File: INSTALL.rst (hunk 1 above)
Comment (anntzer) [inline INSTALL.rst]: Yes (although tkagg may fail if your distro packages python-tk separately and you don't have it installed).

**Comment by choldgraf** [comment]
ah ok...I can't help w/ that then :-)

File: INSTALL.rst (hunk 1 above)
Comment (tacaswell) [inline INSTALL.rst]: ps is more lightly tested than the others, but it has non-zero coverage.

**Comment by QuLogic** [comment]
> ah ok...I can't help w/ that then :-)

You can try liking [this feature request](https://discuss.circleci.com/t/show-test-results-for-prospective-merge-of-a-github-pr/1662) so that CircleCI will start building merges instead of PR head...

File: INSTALL.rst [hunk 2]
```diff
@@ -76,8 +53,31 @@ or
 `32 bit <https://www.microsoft.com/en-us/download/details.aspx?id=5555>`__
 for Python 3.4) redistributable packages need to be installed.
 
+OSX
```
Comment (QuLogic) [inline INSTALL.rst]: Should be `macOS` to match above.

File: INSTALL.rst [hunk 3]
```diff
@@ -76,8 +53,31 @@ or
...
+---
+
+If you are using python 2.7 on a Mac you may need to do::
```
Comment (QuLogic) [inline INSTALL.rst]: Python

File: INSTALL.rst [hunk 4]
```diff
@@ -76,8 +53,31 @@ or
...
+
+To use the native OSX backend you will need :ref:`a framework build
+<osxframework-faq>` build of Python.
+
 
-The Windows wheels (:file:`*.whl`) on the `PyPI download page
+Linux
+-----
+
+On extremely old versions of linux and python 2.7 you may need to
```
Comment (QuLogic) [inline INSTALL.rst]: Linux; Python

**Comment by tacaswell** [comment]
Thanks @NelleV !

**Pull request merged**